### PR TITLE
[docs] - supercollide/supercollider - supercollider/HelpSource/Refere…

### PR DESCRIPTION
**Pull request by tem44**
#### [docs] - supercollide/supercollider - supercollider/HelpSource/Reference/Literals.schelp

### CODE REVIEW

File: HelpSource/Reference/Literals.schelp [hunk 1]
```diff
@@ -67,7 +67,7 @@ inf   // and beyond!
 subsection::Radix
 Numbers can also be written in radices other than base 10 up to base 36.
 The radix is specified in base 10, followed by the letter 'r', followed by the
-value written in that radix using characters 0-9 and A-Z (or a-z) for digit
+the value is written in that radix using characters 0-9 and A-Z (or a-z) for digit
```
Comment (telephon) [inline HelpSource/Reference/Literals.schelp]: maybe better: `The radix is specified in base 10, followed by the letter 'r', followed by the
value. The value is written in that radix […]`

Comment (snappizz) [inline HelpSource/Reference/Literals.schelp]: this was fine, now it has a duplicate "the".

File: HelpSource/Reference/Literals.schelp [hunk 2]
```diff
@@ -94,7 +94,7 @@ code::
 Hexidecimal numbers notated with code::0x:: may only be expressed as integers.
 
 subsection::Scale Degrees
-Integer numbers as scale degrees supports accidentals notation by adding the suffixes strong::s:: for sharp and strong::b:: for flat. Accidentals are represented as floating point values.
+Integer numbers as scale degrees support accidentals notation by adding the suffixes strong::s:: for sharp and strong::b:: for flat. Accidentals are represented as floating point values.
```
Comment (telephon) [inline HelpSource/Reference/Literals.schelp]: I think the `as` is not logically correct. 
Maybe `Integer numbers. Scale degrees support accidentals notation …`.

**Comment by tem44** [comment]
@telephon @snappizz  please double check my corrections. thank you so much

File: HelpSource/Reference/Literals.schelp [hunk 3]
```diff
@@ -66,8 +66,8 @@ inf   // and beyond!
 
 subsection::Radix
 Numbers can also be written in radices other than base 10 up to base 36.
-The radix is specified in base 10, followed by the letter 'r', followed by the
-the value is written in that radix using characters 0-9 and A-Z (or a-z) for digit
+maybe better: The radix is specified in base 10, followed by the letter 'r', followed by the value.
```
Comment (telephon) [inline HelpSource/Reference/Literals.schelp]: There's a copy paste error: "maybe better" wasn't part of my suggestion …

**Comment by tem44** [comment]
hey brian. didn't have time so far to check this. i took a look into it. fixed it. feel free to double check it and please let me know if there's something that needs to be fixed. thanks

**Comment by tem44** [comment]
hey everyone. once I opened the PR, I should fix everything by myself. but in fact, I am having trouble fixing the indentation. I am trying to do it from the browser. I will try again later on this afternoon with the supercollider IDE and then update the PR. 

looking forward

**Comment by brianlheim** [comment]
Closing due to staleness.

**Pull request closed without merging**